public/static/index.html

```diff
@@ -36,6 +36,9 @@
       integrity="sha384-hWVjflwFxL6sNzntih27bfxkr27PmbbK/iSvJ+a4+0owXq79v+lsFkW54bOGbiDQ"
       crossorigin="anonymous"
     />
+    <link href="https://fonts.googleapis.com/css2?family=Montserrat:wght@400;500;600&family=Source+Sans+Pro:wght@400;600;700&display=swap" rel="stylesheet">
```

Comment:
might have issues with this for users in china, right @Nebulis ? how did you fix it in opencerts?

Reply:
Didn't fix this yet... Im not sure we have Google font however (it's not mentioned in the blocked scripts)